### PR TITLE
tests: combine similar network driver tests

### PR DESCRIPTION
**Pull request by kaspar030**
All those tests basically link in the driver and most parts of gnrc.
This PR combines them into one test in order to reduce the amount of redundant compilation.

### CODE REVIEW

**Comment by kYc0o** [comment]
Let's postpone this for the next release.

**Comment by miri64** [comment]
Postponed due to feature freeze

**Comment by OlegHahm** [comment]
This got postponed for already two releases without a real cause as far as I can see - all the more reason to get it merged for this release.

File: tests/driver_netdev/Makefile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+# include specific driver makefiles in order to test compilation
+# in order to make the test application work, comment all but one
+include Makefile.enc28j60
+include Makefile.encx24j600
+include Makefile.kw2xrf
+include Makefile.xbee
+
+# test if multiple drivers are compiled in
+ifneq (x,$(DRIVERS))
```
Comment (miri64) [inline tests/driver_netdev/Makefile]: This always seem to be the case, when I parse the included makefiles correctly. Please explain.

**Comment by miri64** [comment]
Could also benefit of some README and/or test script.

**Comment by kaspar030** [comment]
No time to work on this.

**Pull request closed without merging**